src/app/services/sms.service.ts

```diff
@@ -229,6 +235,23 @@ const sendVerificationOtp = async (
   return send(twilioData, otpData, recipient, message, SmsType.verification)
 }
 
+const sendAdminContactOtp = async (
+  recipient: string,
+  otp: string,
+  userId: string,
+  defaultConfig: TwilioConfig,
+) => {
+  logger.info(`sendAdminContactOtp: userId="${userId}"`)
```

Comment:
might it be better to log in json format? so much easier to parse

Reply:
Example? But yea.... we should have a specific format for how we log things

Reply:
`logger.info({ action: 'sendAdminContactOtp', userId })`

Reply:
Please make an issue, we should do another pass on logging and fix on a layout. Will probably do this in a different PR

Reply:
Made a PR #199